components/core/MarkdownFrame.js

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import { css } from "@emotion/react";
+import "isomorphic-fetch";
+
+import * as Constants from "~/common/constants";
+import { Markdown } from "~/components/system/components/Markdown";
+import { H1, H2, H3, H4, P, UL, OL, LI, Link } from "~/components/system/components/Typography";
+
+const STYLES_ASSET = css`
+  padding: 120px calc(32px + 16px + 8px);
+  position: relative;
+  width: 100%;
+  height: 100%;
+  overflow-y: scroll;
+  will-change: transform;
+  color: ${Constants.system.white};
+`;
+
+const STYLES_BODY = css`
+  width: 100%;
+  max-width: 687px;
```

Comment:
Where does `687px` come from?

Reply:
I used it as a way to ensure we have a maximum of 70 characters in one line